backend/app/views/spree/admin/store_credits/_update.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if success = flash.discard(:success) %>
```

Comment:
Or maybe we should check whether it's possible to use the default handling for `flash` here?